### DISCUSSION TITLE
Add middleware to dataTable parser?

### DISCUSSION THREAD

**Discussion by KaiWissel** [badeball/cypress-cucumber-preprocessor#1024] [General]
Hi folks,

I have a dataTable as follows
| id                                                        | name                  |
| Static_Prefix_<CREATE_RANDOM_ID> | a beautiful name |

I've created a util function which scans any attributes of the dataTable for <CREATE_RANDOM_ID> and replaces it by a random value. There are also some more shenanigans so that I can reuse this random value in later steps. But that's not the point for now.

Currently I've to call my util function explicit on any particular dataTable. What I want to achieve is, to add this function to the general parser of the dataTable. Is there any point where I can install my function as a middleware or would be the only option to overwrite data_table.js?

**Comment by badeball** [comment]
No such thing exist.
